telemetry/test_convert.py

```diff
@@ -291,6 +291,86 @@ def test_normal(self):
             self.assertEqual(converted["histograms"][h], expected_converted_histograms[h])
         self.assertIs(converted["info"].get("geoCountry"), None)
 
+    def print_byte_range(self, data, start=None, end=None):
+        if start is None:
+            start = 0
+        if end is None:
+            end = len(data)
+        for i in range(start, end):
+            pieces = {
+                "i": i,
+                "o": ord(data[i]),
+                "c": data[i]
+            }
+            print "0x%(i)04x %(i)04d = 0x%(o)02x %(o)3d %(c)s" % pieces
+        print "----"
+
+    def test_utf8(self):
+        count = 0
+        # This packed file contains the same record twice, once gzipped, once
+        # raw. The record contains the UTF-8 encoded string "Wikipédia"
+        for r in fu.unpack('test/unicode.v1.packed', file_version="v1"):
+            self.assertIs(r.error, None)
+            self.assertTrue(len(r.data) > 0)
+            self.assertEqual(r.data[0], '{')
+            count += 1
+
+            # self.print_byte_range(r.data, 3323, 3362)
+
+            # Make sure that the raw bytes are correct:
+            # Incoming is UTF-8, so we expect
+            #   W i k i p 0xc3 0xc9 d i a
+            self.assertEqual(ord(r.data[0xd16]), 0x70)
```

Comment:
I used hex indices because I started off by using vim in hex mode to find the expected byte values, so it was easier to match it up.  
I decided not to use literal strings for checking the byte values to avoid any further annoyances with python strings, encodings, bytes, etc.